$OpenBSD: patch-Source_NSException_m,v 1.1 2011/07/19 06:29:09 sebastia Exp $

Fix "hang" when trying to catch exceptions, ugly hack, but fixes a lot
of problems in different applications.

--- Source/NSException.m.orig	Sat Jul 16 14:13:45 2011
+++ Source/NSException.m	Sat Jul 16 14:14:09 2011
@@ -957,7 +957,7 @@ callUncaughtHandler(id value)
     {
       _reserved = NSZoneCalloc([self zone], 2, sizeof(id));
     }
-  _e_stack = [GSStackTrace new];
+  //_e_stack = [GSStackTrace new];
 
 #if     defined(_NATIVE_OBJC_EXCEPTIONS)
   @throw self;
